apps/css-workshop/src/components/Button.astro

```diff
@@ -0,0 +1,22 @@
+---
+type Props = {
+  variant?: 'high-visibility' | 'borderless' | 'cta';
+  size?: 'small' | 'large';
+  active?: boolean;
+  shift?: 'left' | 'right';
+  iuiType?: 'feedback' | 'dropdown';
```

Comment:
This `iuiType` prop doesn't make any sense.
- The `iui-` prefix is only used as a form of namespacing in CSS and DOM elements.
- dropdown and ideas button have nothing to do with each other and shouldn't be grouped this way

Reply:
The `iuiType` prop was added to bring the properties in line with the `data-iui-type` attribute being added in this [PR](https://github.com/iTwin/iTwinUI/pull/1919). If you don't think that the props on the component should line up with the attributes like this then it can be removed.

Reply:
Generally, there isn't a strong reason for components props to mimick how the CSS is implemented. Pick the abstractions that make more sense.

That being said, I left a [comment](https://github.com/iTwin/iTwinUI/pull/1919#discussion_r1567596614) in Jon's PR, because the CSS implementation also feels questionable to me.

Reply:
Okay, will remove.